test/AutoRest.TestServerLowLevel.Tests/dpg_update1.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core;
+using AutoRest.TestServer.Tests.Infrastructure;
+using dpg_update1_LowLevel;
+using NUnit.Framework;
+
+namespace AutoRest.TestServer.Tests
+{
+    public class DpgUpdate1Test : TestServerLowLevelTestBase
+    {
+        [Test]
+        public Task DPGAddOptionalInput() => Test(async (host) =>
+        {
+            var result = await new ParamsClient(Key, host).GetRequiredAsync("a");
+            var responseBody = JsonData.FromBytes(result.Content.ToMemory());
+            Assert.AreEqual("An object was successfully returned", (string)responseBody["message"]);
+            var result2 = await new ParamsClient(Key, host).GetRequiredAsync("a", default);
+            var responseBody2 = JsonData.FromBytes(result2.Content.ToMemory());
+            Assert.AreEqual("An object was successfully returned", (string)responseBody2["message"]);
+            var result3 = await new ParamsClient(Key, host).GetRequiredAsync("a", context: default);
+            var responseBody3 = JsonData.FromBytes(result3.Content.ToMemory());
+            Assert.AreEqual("An object was successfully returned", (string)responseBody3["message"]);
+            var result4 = await new ParamsClient(Key, host).GetRequiredAsync("a", "b");
```

Comment:
Could you help me understand what's happening here?  How do we know we didn't make breaking changes between the initial test and the update1 test?

Reply:
Customized code like overload methods are added to make v2(update1) backward compatible for non-breaking changes from swagger. Here all the test cases of initial are copied to update1. Therefore, we can consider update1 is backward compatible with inital.